lib/msf/ui/console/command_dispatcher/developer.rb

```diff
@@ -51,6 +51,18 @@ def reload_file(path)
     load path
   end
 
+  def reload_diff_files
+    output = %x(git diff --name-only 2>&1)
+    if output.include?('Not a git repository')
+      print_error 'No git repository found.'
+      return
+    end
+    files = output.split("\n")
+    files.each do | file |
```

Comment:
I'd put this on one line, but that's my personal style.

```
files.each { |file| reload_file(file) }
```

If you choose to keep this (and it already prints in `reload_file`, so we're covered there), I'd at least change it to `|files|` (no spaces) as per typical style.